package.json

```diff
@@ -22,6 +22,7 @@
     "babel-polyfill": "^6.26.0",
     "babel-preset-latest": "^6.24.1",
     "babel-register": "^6.26.0",
+    "bcrypt": "^1.0.3",
     "bcryptjs": "^2.4.3",
```

Comment:
Kindly remove the `bcryptjs` package since you are no longer using it in the application